tests/byzantium/eip198_modexp_precompile/test_modexp.py

```diff
@@ -199,6 +199,16 @@ class ExpectedOutput(TestParameterGroup):
             ),
             id="EIP-198-case5-raw-input",
         ),
+        pytest.param(
+            ModExpRawInput(
+                raw_input="ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffa0",
+            ),
+            ExpectedOutput(
+                call_return_code="0x00",
```

Comment:
@marioevz would have been nice to be able to check the exception (oog/stack overflow etc)